test/util/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
```

Comment:
Can you add a crate-level comment that explains its purpose? Doesn't need to be long!

```rust
//! `test_util` provides utilities for test programs and demos that run against Materialize.
```